app/serializers/searched_user_serializer.rb

```diff
@@ -0,0 +1,19 @@
+class SearchedUserSerializer
+include JSONAPI::Serializer
+
+  attributes :first_name, :last_name, :is_remote, :skills
```

Comment:
Nice! You used this serializer to send back specific data for a searched user instead of sending data that won't be needed, looks good!